### PR TITLE
Quickstart code in docs fixed

### PR DESCRIPTION
**Pull request by AlexejPenner**
## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [x] If my change requires a change to the documentation, I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Describe changes
Code was actually failing, reformulated a bit, removed mention of git

### CODE REVIEW

File: docs/book/introduction/quickstart-guide.md [hunk 1]
```diff
@@ -74,33 +79,32 @@ def trainer(
...
+    loss, acc = model.evaluate(x_test, y_test, verbose=1)
+    return loss, acc
 
 
 @pipeline
 def mnist_pipeline(
-    importer,
-    trainer,
-    evaluator,
+        importer,
```
Comment (htahir1) [inline docs/book/introduction/quickstart-guide.md:100]: I think these are extra indents by mistake?

File: docs/book/introduction/quickstart-guide.md [hunk 2]
```diff
@@ -74,33 +79,32 @@ def trainer(
...
-    model.fit(X_train, y_train)
+    model.fit(x_train, y_train)
 
-    # write model
     return model
 
 
 @step
 def evaluator(
-    X_test: np.ndarray,
```
Comment (htahir1) [inline docs/book/introduction/quickstart-guide.md:85]: I actually really liked the capitalized `X_test` for X matrices. I think its a convention (or atleast that how I learned it)

**Review by htahir1 — COMMENTED** [review]
Again thanks for the boyscout work! Just added a few comments

File: docs/book/introduction/quickstart-guide.md (hunk 2 above)
Comment (AlexejPenner) [reply docs/book/introduction/quickstart-guide.md:85]: Alright, I can return it to capitalized

**Comment by AlexejPenner** [comment]
@htahir1 I addressed the issues you raised :)

**Review by htahir1 — APPROVED** [review]
LGTM!!!

**Review by AdamVPro — APPROVED** [review]
what Hamza said

**Pull request merged**